src/System.Collections.Immutable/ref/System.Collections.Immutable.csproj

```diff
@@ -4,14 +4,9 @@
   <PropertyGroup>
     <ProjectGuid>{C7EFF4EE-70DC-453B-B817-4AF67921AB03}</ProjectGuid>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(TargetGroup)' != 'netstandard1.0'">
+    <DefineConstants>$(DefineConstants);ItemRefApi</DefineConstants>
```

Comment:
Rather than `ItemRefApi` I suggest `NETSTANDARD1_3`. This is the format the new SDK uses when doing multi-targeting and is likely to be more familiar to users. Not sure if that's the standard in this repo though.

Reply:
See https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#define-naming-convention which contains our naming conventions for these defines. They should match the configuration they are intended for.